Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -92,7 +94,8 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
 //            assert(Thread.isMainThread)
 //
 //        }.store(in: &cancellables)
-
+        
+        setupEditor()
```

Comment:
Small grammatical improvement. Feel free to ignore, if you prefer. 
```suggestion
        setUpEditor()
```

Reply:
Updated in https://github.com/wordpress-mobile/GutenbergKit/pull/15/commits/29324052a16a172141e0821c789cc0c0595b7fcd